packages/server/assets/core/typedefs/projects.graphql

```diff
@@ -30,6 +30,7 @@ input ProjectCreateInput {
   name: String
   description: String
   visibility: ProjectVisibility
+  workspaceId: String
```

Comment:
I'm a bit unsure about this, maybe we'll split the project creation resolver for workspaces into the workspaces schema. For now this is ok, but we should go through all the licensing implications.

Reply:
Leaning more towards the split more, but also in agreement that this can wait